repositories/org_repository.go

```diff
@@ -34,6 +46,27 @@ func NewOrgRepo(rootNamespace string, privilegedClient client.Client) *OrgRepo {
 	}
 }
 
+func (r *OrgRepo) CreateOrg(ctx context.Context, org OrgRecord) (OrgRecord, error) {
+	anchor := &v1alpha2.SubnamespaceAnchor{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: org.Name,
+			Namespace:    r.rootNamespace,
+			Labels: map[string]string{
+				OrgNameLabel: org.Name,
+			},
+		},
+	}
+	err := r.privilegedClient.Create(ctx, anchor)
+	if err != nil {
+		return OrgRecord{}, err
+	}
+
+	org.GUID = string(anchor.UID)
```

Comment:
this should be `anchor.Name` based on the change to use google's uuid

Reply:
Done